src/app/core/common-components/entity-subrecord/entity-subrecord/entity-subrecord.component.spec.ts

```diff
@@ -184,24 +183,6 @@ describe("EntitySubrecordComponent", () => {
     expect(sortedNames).toEqual(["A", "b", "C"]);
   });
 
-  it("should log a warning when the column definition can not be initialized", () => {
```

Comment:
This does not seem to have been handled in subrecord component anyway? Was this supposed to come from appDynamicComponent?

Reply:
yes, there is a catch, which could catch the error thrown by EntityFormService.extendFormFieldConfig  But actually I do not see how any error could occur in the EntityFormService in the current implementation. Do you? Otherwise let's remove both try-catch blocks and only rely on the errors for broken field config in the individual FormFieldComponent?